### ISSUE TITLE
Highlight.js language support

### ISSUE THREAD

**Issue by shekhirin** [ethereum/fe#554]
### What is wrong?

Currently even https://fe-lang.org/ uses `python` language for highlight.js library syntax highlighting. It may be handy for educational and other Fe-related resources to use dedicated syntax highlighter for Fe. 

### How can it be fixed

Contribute Fe language support to [hightlight.js](https://github.com/highlightjs/highlight.js). Good reference is a [Python implementation](https://github.com/highlightjs/highlight.js/blob/main/src/languages/python.js).

**Comment by cburgdorf** [comment]
Yes, we absolutely want that. Would you be interested to work on that yourself?

**Comment by shekhirin** [comment]
@cburgdorf yep, that sounds useful for the community and quite easy to do, however I'd wait a bit until language grammar stabilizes so we won't need to update Highlight.js support every release.
